src/util/JsonUtils.hpp

```diff
@@ -112,26 +115,47 @@ integralValueAs(boost::json::value const& value)
     throw std::logic_error("Value neither uint64 nor int64");
```

Comment:
Could you please use `tryIntegralValueAs()` here and throw exception if it returned unexpected?